src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/BooleanExpressionComplexityCheck.java

```diff
@@ -53,6 +53,7 @@
  * Property {@code max} - Specify the maximum number of boolean operations
  * allowed in one expression.
  * Default value is {@code 3}.
+ * Datatype - {@code integer}
```

Comment:
Let's do just `Type` as it is human readable content. In xml we can use another name.

Reply:
But @strkkk asked to make it consistent with https://checkstyle.org/property_types.html#Overview. It is `integer` in it.

Reply:
Eventually xdoc and javadoc should be exactly the same. I feel if xdoc says Integer, javadoc should say the same. To me "Integer" is a more user friendly display than the shorthand "int" which is a more technical type that users may not understand.

Reply:
yes, lets do Integer as we mentioned in xdoc description.

Reply:
Done.